### DISCUSSION TITLE
Height and Width properties

### DISCUSSION THREAD

**Discussion by smfields** [Megabit/Blazorise#4687] [Q&A]
I'm a little bit confused on how the Height and Width properties are supposed to work? What is the intended way to specify a custom height (such as in px) on an component?

**Comment by David-Moreira** [comment]
> I'm a little bit confused on how the Height and Width properties are supposed to work? What is the intended way to specify a custom height (such as in px) on an component?

Hello,
The height and width as well as other helpful parameters have a strongly typed fluent helper associated with them with pre defined options.
If you want to get very specific with it, then just using classes or style is the way to go by using the Class or Style parameters.

**Reply by stsrki** [reply]
They are based on Bootstrap utilities. Please take a look at https://getbootstrap.com/docs/5.3/utilities/sizing/ for reference.
